inspirehep/records/api/literature.py

```diff
@@ -80,7 +85,8 @@ def add_files(self, documents=None, figures=None):
         """Public method for adding documents and figures
 
         Args:
-            documents (list[dict]): List of documents which should be added to this record
+            documents (list[dict]): List of documents which should be added to this
```

Comment:
`list` -> `List` as `list == List[Any]`.

Reply:
[Sphinx Example Google Style Python Docstrings](https://sphinx.readthedocs.io/en/master/usage/extensions/example_google.html):
`
    param3 (list(str)): Description of 'param3'.
`